src/i18n/de/nav.ts

```diff
@@ -18,11 +20,32 @@ export default NavDictionary({
 	'core-concepts/astro-components': 'Komponenten',
 	'core-concepts/astro-pages': 'Seiten',
 	'core-concepts/layouts': 'Layouts',
+	'guides/markdown-content': 'Markdown',
+	'core-concepts/routing': 'Routing',
+	'guides/imports': 'Statische Inhalte',
+	'guides/troubleshooting': 'Fehlerbehebung',
+	'guides/deploy': 'Veröffentlichung',
 
 	// Features
 	features: 'Anleitungen',
+	'guides/configuring-astro': 'Astro konfigurieren',
 	'guides/styling': 'Styling & CSS',
+	'guides/data-fetching': 'Abrufen von Daten',
+	'guides/environment-variables': 'Umgebungsvariablen',
+	'guides/aliases': 'Import-Aliasnamen',
+	'guides/integrations-guide': 'Integrationen',
+	'guides/rss': 'RSS',
+	'guides/server-side-rendering': 'Serverseitiges Rendern',
```

Comment:
Would it make sense to keep the abbreviation here like in the English given we use it in the deploy guides and other places?

```suggestion
	'guides/server-side-rendering': 'Serverseitiges Rendern (SSR)',
```